src/shared/main/java/com/couchbase/lite/LogFileConfiguration.java

```diff
@@ -44,33 +68,18 @@ public LogFileConfiguration(@NonNull String directory) {
      *
      * @param other The other configuration to copy settings from
      */
-    public LogFileConfiguration(@NonNull LogFileConfiguration other) {
-        if (other == null) {
-            throw new IllegalArgumentException("other cannot be null");
-        }
+    private LogFileConfiguration(@NonNull LogFileConfiguration other, boolean readonly) {
+        if (other == null) { throw new IllegalArgumentException("other cannot be null"); }
```

Comment:
instead of `"other cannot be null"`, i feel better to throw as `"config cannot be null"`.

Reply:
good call. Much clearer.
Will do.